bigbluebutton-html5/private/locales/en.json

```diff
@@ -196,7 +196,7 @@
     "app.audioModal.microphoneLabel": "Microphone",
     "app.audioModal.listenOnlyLabel": "Listen Only",
     "app.audioModal.audioChoiceLabel": "How would you like to join the audio?",
-    "app.audioModal.audioChoiceDescription": "Select how to join the audio in this meeting",
+    "app.audioModal.audioChoiceDesc": "Select how to join the audio in this meeting",
```

Comment:
I can't find where we use this description in the client. Maybe we don't need it here at all?